homeassistant/components/anova/coordinator.py

```diff
@@ -17,37 +15,22 @@
 class AnovaCoordinator(DataUpdateCoordinator[APCUpdate]):
     """Anova custom coordinator."""
 
-    def __init__(
-        self,
-        hass: HomeAssistant,
-        anova_device: AnovaPrecisionCooker,
-    ) -> None:
+    def __init__(self, hass: HomeAssistant, anova_device: APCWifiDevice) -> None:
         """Set up Anova Coordinator."""
         super().__init__(
             hass,
             name="Anova Precision Cooker",
             logger=_LOGGER,
-            update_interval=timedelta(seconds=30),
         )
         assert self.config_entry is not None
```

Comment:
You can remove this check by adding `config_entry: ConfigEntry` outside of the constructor to retype the variable